jbmc/src/java_bytecode/java_object_factory.cpp

```diff
@@ -17,6 +17,7 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <goto-programs/class_identifier.h>
 #include <goto-programs/goto_functions.h>
+#include <solvers/strings/string_constraint_generator.h>
```

Comment:
I'm not sure it's a good thing to have a dependency of java bytecode on solvers (@peterschrammel?). A solution for that would be to move the function you need to `util`, but make it take a lower bound and an upper bound instead of a string, rename it `interval_constraint` or something similar, and make it work for more types (it should take mp_integers as inputs).

Reply:
Yes, that's preferable.